eng/build.sh

```diff
@@ -377,6 +378,18 @@ while [[ $# > 0 ]]; do
       shift 2
       ;;
 
+     -keepsymbols)
+      if [ -z ${2+x} ]; then
+        echo "No value for keep symbols is supplied. See help (--help) for supported values." 1>&2
+        exit 1
+      fi
+      passedKeepSymbols="$(echo "$2" | awk '{print tolower($0)}')"
+      if [ "$passedKeepSymbols" = true ]; then
+        cmakeargs="${cmakeargs} -cmakeargs -DCLR_KEEP_SYMBOLS=true"
```

Comment:
A nit - based on various cmake args settings in coreclr, I'd prefer this to be named CLR_CMAKE_KEEP_SYMBOLS to keep things unified.

Reply:
Done!